README.md

```diff
@@ -1,4 +1,60 @@
-node-voicemail-admin
-====================
+# Asterisk Voicemail Administrator Application
+
+A command line administration tool for managing mailboxes in Asterisk voicemail
+
+# Installation
+
+```bash
+$ git clone https://github.com/asterisk/node-voicemail-admin.git
+$ cd node-voicemail-admin
+$ npm install -g .
```

Comment:
I agree with Sam. -g is good for things like installing grunt or yeoman, where you may want to be able to easily use it from your command prompt/shell/terminal. 

Of course, you can still install node packages locally, even if they are installed globally.

http://blog.nodejs.org/2011/03/23/npm-1-0-global-vs-local-installation
https://docs.npmjs.com/misc/faq